focus-ios/Blockzilla/Utilities/URIFixup.swift

```diff
@@ -35,11 +35,11 @@ class URIFixup {
         // make sure there's at least one "." in the host. This means
         // we'll allow single-word searches (e.g., "foo") at the expense
         // of breaking single-word hosts without a scheme (e.g., "localhost").
-        if trimmed.range(of: ".") == nil {
+        if trimmed.contains(".") == false {
             return nil
         }
 
-        if trimmed.range(of: " ") != nil {
+        if trimmed.contains(" ") == true {
```

Comment:
Nit: for style consistency we should probably remove this `== true`, and I believe we generally prefer `!<condition>` over `<condition> == false` but otherwise LGTM.

Reply:
I was wondering what the preferred style was.  Thanks for letting me know!